samples/crypto/aes_gcm/boards/nrf54l15dk_nrf54l15_cpuapp.conf

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+CONFIG_PSA_CRYPTO_DRIVER_OBERON=n
+CONFIG_PSA_CRYPTO_DRIVER_CRACEN=y
```

Comment:
Do we actually need those? (I would vote for leaving them out if not.)

Reply:
(Need in the sense they're not enabled by default. Because I imagine we do want to use HW crypto.)

Reply:
Not sure if it must be there, it has just been the default config for 54l15 target for crypto samples for some time.
Let's not fix it now in this PR since these samples are failing on main.